include/EASTL/stack.h

```diff
@@ -115,7 +115,7 @@ namespace eastl
 		void push(value_type&& x);
 
 		template <class... Args>
-		void emplace_back(Args&&... args);
+		void emplace(Args&&... args);
```

Comment:
We should maintain backwards compat and give users an opportunity to migrate off the old API.

Reply:
Does EASTL target C++14 as minimum C++ standard. If so we can add `[[deprecated]]` to the old one to make it explicitly?

Reply:
Yes.  We have EA_DEPRECATED portability macros to enable/disable that... or use other features of compiler if that doesn't work.